lib/std/decode_helpers.nim

```diff
@@ -1,13 +1,11 @@
-# Include file that implements 'decodePercent' and friends. Do not import it!
```

Comment:
rename it to `lib/std/private/decode_helpers.nim`, that way it discourages people to import this and buys more time to "fix the API" if needed

Reply:
I agree.